analyzer/src/main/java/org/jboss/gm/analyzer/alignment/DAAlignmentService.java

```diff
@@ -37,7 +38,9 @@ public DAAlignmentService(Configuration configuration) {
 
         final String endpointUrl = configuration.daEndpoint();
 
-        if (endpointUrl == null && dependencySource != NONE) {
+        logger.info("endpointUrl = {}, dependencySource = {}", endpointUrl, dependencySource);
```

Comment:
Perhaps debug level logging?

Reply:
I tried info, debug, and println and I am not seeing it in the log. It's like it's not being called.

Reply:
I am seeing it if I run manually ; remember Gradle has lifecycle logging as well which is the default

Reply:
I was checking in the HTML produced after a build failure (the FIXME in `MainTest`). I do see the line from another method "DAAlignmentService.java:51 No dependencySource configured ; unable to call endpoint?", but not this one.

That line implies `dependencySource == NONE` which won't trigger the `if` which has `dependencySource != NONE`, although the test seems to expect it to trigger.